pkgs/applications/science/electronics/kicad/default.nix

```
@@ -11,13 +13,16 @@
 , callPackages
 , librsvg
 , cups
+, unzip
+, jq
 
 , pname ? "kicad"
 , stable ? true
 , withNgspice ? !stdenv.isDarwin
 , libngspice
 , withScripting ? true
 , python3
+, thirdPartyPackages ? [ ]
```

```suggestion
, extraPythonPackages ? [ ]
```
or something similar to this

---

We actually chose the phrase "third party" intentionally. It seems to be a term from kicad that refers to "footprints", "plugins", "resources", or "symbols". Kicad's Plugin and Content Manager (PCM) installs stuff to `~/.local/share/kicad/7.0/3rdparty/`, and the environment variable to configure that location is `KICAD7_3RD_PARTY`.